### PR TITLE
Update IBPSA rc parameter names and matrix dimension

### PR DESCRIPTION
**Pull request by nllong**
The versions of the RC model in IBPSA and in the Modelica Building Library have updated parameter names (e.g., hConvExt is now hConExt). In addition, the dimension of the internal gain matrices are incorrect.

This PR accomplished the following:
* fixes the names of the parameters in the IBPSA RC models
* fixes the size of the internal gain matrix for the IBPSA models. 
* Adds a new first data row with a timestep of zero to the internal gain files
* Catch if institute4 does not have central_air

Please let me know if you want me to create issues. Also, I'm unable to assign a reviewer, presumably due to permissions. @PRemmen 

Local tests were run with the following result:
```
=========== 348 passed, 130 warnings in 179.16s (0:02:59) ===================
```

### CODE REVIEW

File: teaser/logic/archetypebuildings/bmvbs/custom/institute4.py [hunk 1]
```diff
@@ -177,4 +177,5 @@ def __init__(self,
         self.est_exponent_wall = 0.6177
         self.est_factor_wall_area = 13.895
 
-        self.central_ahu.humidification = False
+        if self.central_ahu:
```
Comment (nllong) [inline teaser/logic/archetypebuildings/bmvbs/custom/institute4.py:180]: The central AHU can be uninitialized in this archetype building.

File: teaser/logic/buildingobjects/calculation/ibpsa.py [hunk 2]
```diff
@@ -114,11 +114,10 @@ def modelica_gains_boundary(self, zone, path=None):
...
             f.write("#1\n")
-            f.write(
-                "double Internals({}, {})\n".format(
-                    8760, (len(self.parent.thermal_zones) * 3 + 1)
-                )
-            )
+            # The size of the dataset is always 4 columns as each thermal zone has its own data file.
+            f.write("double Internals(8761, 4)\n")
+            # write the first row with t=0
+            f.write("0\t{}".format(export.iloc[:1, :].to_csv(sep="\t", header=False, index=False)))
```
Comment (nllong) [inline teaser/logic/buildingobjects/calculation/ibpsa.py:120]: File looks like:
```
#1
double Internals(8761, 4)
0	0.0	0.0	10701.417762000003
3600	0.0	0.0	10701.417762000003
...
```

**Comment by PRemmen** [comment]
@MartinRaetz @martinkremer could you have a look at this PR :)?

@nllong thanks a lot for the PR! :) A short issue would be great.

**Comment by PRemmen** [comment]
as both reviewers accepted this merge request: I'll merge

Thanks @nllong !

**Comment by nllong** [comment]
Great, thanks everyone!

**Pull request merged**